src/argilla/client/feedback/integrations/huggingface/dataset.py

```diff
@@ -160,10 +162,10 @@ def _huggingface_format(dataset: Union["FeedbackDataset", "RemoteFeedbackDataset
             hf_dataset["external_id"].append(record.external_id or None)
 
             if vectors_settings:
+                vectors = {}
                 for vector_settings in vectors_settings:
-                    if vector_settings.name not in hf_dataset:
-                        hf_dataset[vector_settings.name] = []
-                    hf_dataset[vector_settings.name].append(record.vectors.get(vector_settings.name, None))
+                    vectors.update({vector_settings.name: record.vectors.get(vector_settings.name, None)})
```

Comment:
Here we can have a problem when loading the dataset from huggingface with `None`. After loading this, when pushing to argilla, the vector validation will raise an error with a `None` value that is not a valid `list[float]` or something similar.

Reply:
```suggestion
                    vector_value = record.vectors.get(vector_settings.name, None)
                    if vector_value:
                       vectors.update({vector_settings.name: vector_value})
```